backend/scraper/scrape_target.py

```diff
@@ -0,0 +1,148 @@
+import bs4 as bs
+import sys
+import urllib
+from PyQt5.QtWebEngineWidgets import QWebEnginePage
+from PyQt5.QtWidgets import QApplication
+from PyQt5.QtCore import QUrl
+
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+import re
+import time
+import requests
+
+PATH = "A:\chrome driver\chromedriver.exe"
+driver = webdriver.Chrome(PATH)
```

Comment:
Wait, so you still need Selenium? I thought PyQtWebEngine was taking care of that?